samples/bluetooth/cap_acceptor/src/cap_acceptor_broadcast.c

```diff
@@ -568,6 +568,13 @@ static void bap_pa_sync_terminated_cb(struct bt_le_per_adv_sync *sync,
 		atomic_clear_bit(flags, FLAG_BROADCAST_CODE_RECEIVED);
 		atomic_clear_bit(flags, FLAG_BROADCAST_SYNC_REQUESTED);
 
+		err = bt_bap_broadcast_sink_delete(broadcast_sink.bap_broadcast_sink);
+		if (err != 0) {
+			LOG_ERR("Failed to delete Broadcast Sink: %d", err);
+		} else {
+			broadcast_sink.bap_broadcast_sink = NULL;
+		}
```

Comment:
Can you elaborate on this change? 

If we lose the PA sync, do we always want to terminate the BIG sync? 

Would it make sense to, instead, add the `stopped` callback from `bt_bap_broadcast_sink_cb` and delete the broadcast sink object in that?